### PR TITLE
Dev/new packages: added several split packages for wps-office

### PR DESCRIPTION
**Pull request by butangmucat**
Upstream have renamed kingsoft-office to wps-office. Split the packages because they are meant to be split by upstream. Resolved issue #53.

### CODE REVIEW

File: extra-editors/wps-office/autobuild/postinst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+ksv_et_mimes=('application-wps-office.et' 'wps-office-et.png' 'application-wps-office.ett' 'wps-office-ett.png')
+ksv_excel_mimes=('application-wps-office.xls' 'wps-office-xls.png' 'application-wps-office.xlt' 'wps-office-xlt.png' 'application-vnd.ms-excel' 'wps-office-xls.png' 'application-msexcel' 'wps-office-xls.png' 'application-wps-office.xlsx' 'wps-office-xls.png' 'application-wps-office.xltx' 'wps-office-xlt.png' )
+ksv_wpp_mimes=('application-wps-office.dps' 'wps-office-dps.png' 'application-wps-office.dpt' 'wps-office-dpt.png')
+ksv_ppt_mimes=('application-wps-office.ppt' 'wps-office-ppt.png' 'application-wps-office.pot' 'wps-office-pot.png' 'application-vnd.ms-powerpoint' 'wps-office-ppt.png' 'application-vnd.mspowerpoint' 'wps-office-ppt.png' 'application-powerpoint' 'wps-office-ppt.png' 'application-wps-office.pptx' 'wps-office-ppt.png' 'application-wps-office.potx' 'wps-office-pot.png' )
+
+function ks_install_icon()
```
Comment (Arthur2e5) [inline extra-editors/wps-office/autobuild/postinst]: Please, never use the old `function foo` syntax. `foo()` is enough and more portable (as of POSIX). There combination `function foo()` is also not recommended.

Comment (Arthur2e5) [inline extra-editors/wps-office/autobuild/postinst]: Please, never use the old `function foo` syntax. `foo()` is enough and more portable (as of POSIX). Their combination `function foo()` is also not recommended.

File: extra-editors/wps-office/autobuild/postinst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+function ks_main()
+{
+	if [ $# -eq 0 ] ; then
+		return;
+	fi
+
+	ks_init
+	case $1 in
```
Comment (Arthur2e5) [inline extra-editors/wps-office/autobuild/postinst]: Sadly we are still unsure about rpm post/pre scripts.

File: extra-editors/wps-office/autobuild/postinst [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+		configure )
+			shift
+			ks_config $@
```
Comment (Arthur2e5) [inline extra-editors/wps-office/autobuild/postinst]: `"$@"`

File: extra-i18n/wps-office-spellcheck/autobuild/build [hunk 4]
```diff
@@ -0,0 +1,23 @@
+mkdir -p ${PKGDIR}/usr/lib/office6/data/
+wget -c http://wps-community.org/download/dicts/ca_ES.zip
```
Comment (Arthur2e5) [inline extra-i18n/wps-office-spellcheck/autobuild/build]: Never considered a for loop? Btw wget supports multiple URLs, so you can write `wget -c http://wps-community.org/download/dicts/{ca_ES,cs_CZ,blah}.zip`.

File: extra-editors/wps-office/autobuild/postinst (hunk 1 above)
Comment (butangmucat) [inline extra-editors/wps-office/autobuild/postinst]: I believe this script came from upstream. You can do whatever you need to make it better.

File: extra-editors/wps-office/autobuild/postinst (hunk 3 above)
Comment (butangmucat) [inline extra-editors/wps-office/autobuild/postinst]: This is the pot of the upstream.

File: extra-i18n/wps-office-spellcheck/autobuild/build (hunk 4 above)
Comment (butangmucat) [inline extra-i18n/wps-office-spellcheck/autobuild/build]: You can change it, the original script is from Jeff.

File: extra-editors/wps-office/autobuild/postinst (hunk 1 above)
Comment (MingcongBai) [inline extra-editors/wps-office/autobuild/postinst]: @Arthur2e5 This is the official postinst script from Kingsoft, no doubt.

File: extra-i18n/wps-office-spellcheck/autobuild/build (hunk 4 above)
Comment (MingcongBai) [inline extra-i18n/wps-office-spellcheck/autobuild/build]: @Arthur2e5 I will have a separate commit to fix this.

**Comment by MingcongBai** [comment]
Please follow up with later changes. Thanks.

@Arthur2e5 @butangmucat

**Pull request merged**